src/control/ToolEnums.h

```diff
@@ -21,6 +21,7 @@ enum ToolSize {
     TOOL_SIZE_MEDIUM,
     TOOL_SIZE_THICK,
     TOOL_SIZE_VERY_THICK,
+    TOOL_SIZE_CUSTOM,
```

Comment:
Be sure to handle `TOOL_SIZE_CUSTOM` in `ToolEnums.cpp`!

Reply:
Thanks!